wiki/en/Running-a-Server.md

```diff
@@ -36,84 +37,101 @@ However, various problems can also arise when setting up Servers - especially wh
+
+### 1. Unregistered
 
 This is the default when starting a Server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your Server's address to will be able to connect to it. This is useful because Jamulus does not let you control who can connect to a Server.
 
+**For information about running an Unregistered Server [see this guide](Unregistered-Servers).**
+
 <figure>
 	<img src="{% include img/en-screenshots/diagram-unreg-server.inc %}" loading="lazy" alt="Diagram of connections between Clients within a Jamulus Unregistered Server">
-	<figcaption>How Unregistered Servers work</figcaption>
 </figure>
 
-If you are running an unregistered server behind a home internet connection, you might need to enable [port forwarding](#port-forwarding) as described below.
-
-### 3. Directory
-
-If you want to run a number of Servers, possibly also behind a firewall or on a LAN, you may want to run your Server as a Directory. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
+### 2. Registered
 
-To run a Directory [read this guide](Directories)
+Your Server will be listed in a Directory. Jamulus has a list of built-in Directories that Clients can connect to, and if you register your Server with one of these, anyone can then discover and connect to it. You can also have your Server listed on a Custom Directory (see below), if that better meets your requirements.
 
+Note that Directories can only register up to 150 Servers. If you see a message that says you cannot register your Server becuase the Directory is full, you can try registering on another. For a list of all public Directories and the Servers on them, see [explorer.jamulus.io](https://explorer.jamulus.io).
 
-## Installation and Configuration
+<figure>
+	<img src="{% include img/en-screenshots/diagram-reg-server.inc %}" loading="lazy" alt="Diagram of connections between Clients within a Jamulus Registered Server">
+</figure>
 
-Most people run Jamulus as a "pure" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on Windows or on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop).  
+### 3. Directory
 
-If you want to run a Server on a Raspberry Pi (or a different armhf/arm64 debian-based device), you will need to download the `.deb` files for 32 bit `armhf` or 64 bit `arm64`, not the default `amd64` ones you'd use on an Intel/AMD based machine.
+If you want to run a number of Servers, possibly also behind a firewall or on a LAN, you may want to run your Server as a Directory. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
 
+For information, [see the Directories guide](Directories).
 
-### Installation
+# Configuration
```

Comment:
Probably incorrect heading level. This should not be h1.

Reply:
Oh actually, it needs to be H1 or the structure doesn't work. Sorry!

Reply:
H1 should be the highest level headline (what the whole page is about): https://yoast.com/one-h1-heading-per-post/

if it’s just about graphics, we should redesign h2 and lower level headings.

Reply:
It's mainly due to the similar styling I think. Can you remind me which of the .css files we'd need to amend if we did alter these? I always get confused about what gets used for what.